src/main/java/core/basesyntax/service/ParserOperationService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ParserOperationService {
+    List<FruitTransaction> parser(List<String> data);
```

Comment:
Consider renaming the method 'parser' to 'parse' for better readability and convention.